src/components/settings.js

```diff
@@ -0,0 +1,136 @@
+
+import React,{ Component } from 'react';
+import {GridList, GridTile} from 'material-ui/GridList';
+import IconButton from 'material-ui/IconButton';
+
+import ActionDone from 'material-ui/svg-icons/action/done';
+import EditorModeEdit from 'material-ui/svg-icons/editor/mode-edit';
+
+import Avatar from 'material-ui/Avatar';
+import RaisedButton from 'material-ui/RaisedButton';
+
+import HeaderNav from './headerNav';
+
+import TextField from 'material-ui/TextField';
+import '../css/index.css'
```

Comment:
why?

Reply:
liine 12 :-> To use headerNav on the settings page

line 14 :-> To use TextField while editing user email and Names
line 15 :-> To use the index.css on the settings page